README.md

```diff
@@ -93,10 +93,44 @@ You can also follow build statuses in https://percy.io/Homeday/homeday-blocks
 - `tests/` is an alias for `<rootDir>/tests/`
 - The form components share many classes (mostly `field--*`), to avoid repeatedly defining some of them in each form test, [FIELD_CLASSES.js](https://github.com/homeday-de/homeday-blocks/blob/develop/tests/unit/components/form/FIELD_CLASSES.js) contains the most common classes we need for testing. Feel free to add more classes.
 
-## Deployment
+## Contribution guide
 
-The [CI](https://travis-ci.com/homeday-de/homeday-blocks) will deploy storybook automatically to our [showcase env](https://blocks.homeday.dev), once PR is merged to `develop` branch. 
+Project follows [forking workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). See [project setup](#project-setup) to get started locally. That means that all code changes enter the project by PR to `develop` branch. Once you open the PR with suggested changes the checks for `build` and `coverage` will run. If those fail, PR needs some more work. :) 
 
+Each PR should be reviewed by 2 team members. Once reviewed and approved,it can be merged. Please follow the following convention when merging the PR: 
```

Comment:
```suggestion
Each PR should be reviewed by at least two team members. Once reviewed and approved,it can be merged. Please follow the following convention when merging the PR: 
```